lib/player_choice.rb

```diff
@@ -0,0 +1,14 @@
+class PlayerChoice
+    def is_user_input_valid(choice)
+        if choice !~ /\D/ && choice.length > 0
+          num = choice.to_i
+          if num <= 9 && num >= 1
```

Comment:
You could also do a `return num <= 9 && num >= 1` instead of the conditionals

Reply:
Yes, I like that much better!